docs/source/notebook.rst

```diff
@@ -202,12 +211,14 @@ Structure of a notebook document
 
 The notebook consists of a sequence of cells.  A cell is a multi-line
 text input field, and its contents can be executed by using
-:kbd:`Shift-Enter`, or by clicking either the "Play" button the toolbar, or
-`Cell | Run` in the menu bar.  The execution behavior of a cell is determined
+:kbd:`Shift-Enter`, or by clicking either the "Run Cell" button (which looks
```

Comment:
Suggested:

The notebook consists of a sequence of cells.  A cell is a multi-line
 text input field, and its contents can be executed by using
 :kbd:`Shift-Enter`, clicking the "Play" button the toolbar, or selecting `Cell | Run cells` in the menu. The cell's type determines its behavior when executed.